<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodeFileDlg</class>
 <widget class="QMainWindow" name="EncodeFileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="DestFile">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="FileSizeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>BoolLoader固件版本：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>加密文件：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="BootLoaderL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="firmwaretype">
      <item>
       <property name="text">
        <string>X1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D4</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="AppLV">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>升级固件的大小：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>加密文件：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="hardwarev">
      <item>
       <property name="text">
        <string>D2C/D3C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DK1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DKC4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>X1S</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>X1</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="BootLoaderH">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>固件生成世间：</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>固件更新操作人代号：</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="user">
      <item>
       <property name="text">
        <string>ZZZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LDF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MAQ</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>3Glasses</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>固件设备名：</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>保留内容：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>硬件版本信息：</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>应用程序版本：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="SrcFile">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>制造商名：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="AppHV">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>加密</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QTextEdit" name="reserved"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>EncodeFileDlg</receiver>
   <slot>selectFile(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>EncodeFileDlg</receiver>
   <slot>EncodeFile(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFile(bool)</slot>
  <slot>EncodeFile(bool)</slot>
 </slots>
</ui>
